src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+
+- You don't control the color of the chart
+- The sections of the chart have rounded edges
+
+For the sake of this tutorial, imagine this chart represents your data exactly how you want it to, except for two things. You'd like to use straight edges and control the chart's colors manually. In the real world you may come across Nerdpacks like this where you like what they offer, but you'd like them better if you could tweak them. Well, you can tweak them, and next, you'll learn how!
+
+</Step>
+
+<Step>
+
+Because you're going to use a tweaked version of the **Victory Charts** Nerdpack instead of the one you subscribed to, you can [unsubscribe](/build-apps/publish-deploy/subscribe/#unsubscribe-from-a-nerdpack) from our version now.
+
+</Step>
+
+</Steps>
+
+## Clone a Nerdpack
+
+Find the source code repository from the Nerdpack's catalog entry and clone it to your local machine.
+
+<Steps>
+
+<Step>
+
+From your homepage at New Relic, navigate to **Apps**:
+
+![Navigate to apps](../../images/build-an-app/nav-to-apps.png)
+
+</Step>
+
+<Step>
+
+Click the **Victory Charts** Nerdpack in the catalog:
+
+![Victory Charts Nerdpack](../../images/build-an-app/victory-charts-nerdpack.png)
+
+</Step>
+
+<Step>
+
+Go to the Nerdpack's source code repository:
+
+![Victory Charts repo](../../images/build-an-app/victory-charts-repo.png)
+
+All open source Nerdpacks in the catalog have links to their source code in their catalog information.
+
+</Step>
+
+<Step>
+
+Clone the repository:
+
+```sh
+nr1 nerdpack:clone -r https://github.com/newrelic/nr1-victory-visualizations.git
+```
+
+Now you have a local version of the **Victory Charts** Nerdpack! Notice that you used `nr1 nerdpack:clone` instead of `git clone` to copy the repo. `nr1 nerdpack:clone` offers built in functionality to help keep your local copy distinct from the original Nerdpack in the New Relic One public catalog. Specifically, it generates a new Nerdpack UUID so you don't have to do this yourself:
+
+```sh
+[output] Re-generating UUID...
+[output] Committing new UUID...
+```
+
+If you change to the _nr1-victory-visualizations_ directory, and look at the git log, you'll see the new commit:
+
+```sh
+git log -1 -p
+[output] {blue}commit e356bb5b10c3ecc8f93bae66d5739e1676ee21ef {plain}({green}HEAD -> main{plain})
+[output] Author: New Relic CLI <nr1@newrelic.com>
+[output] Date:   Tue May 25 14:29:37 2021 -0400
+[output]
+[output]     "chore: Auto-generated UUID"
+[output]
+[output] diff --git a/nr1.json b/nr1.json
+[output] index 054de52..7a107b5 100644
+[output] --- a/nr1.json
+[output] +++ b/nr1.json
+[output] @@ -1,6 +1,6 @@
+[output]  {
+[output]    "schemaType": "NERDPACK",
+[output] {red}-  "id": "cf5f13d9-815f-4907-a21d-83d02fa2a4fb",
+[output] {green}+  "id": "ab123c45-678d-9012-efg3-45hi6jkl7890",
+[output]    "displayName": "Victory charts visualizations",
+[output]    "description": "Visualizations built on top of Victory charts"
+[output]  }
+```
+
+</Step>
+
+</Steps>
+
```

Comment:
Do you think we should suggest they modify the nerdpack's `displayName` in the `nr1.json` here?

Since they are publishing this nerdpack, encouraging a variation on the name, like `displayName": "Foo Bar's Victory charts visualizations",` should help avoid a case where the app catalog lists multiple tiles for nerdpacks with the same outward appearance, but different IDs.

We've managed to not follow our own advice in staging and this is what it can end up like 😅 

![image](https://user-images.githubusercontent.com/20293876/120049841-25032d00-bfd0-11eb-9ae9-9922fa6be42d.png)